messaging/src/test/java/org/axonframework/deadline/jobrunr/DeadlineDetailsSerializationTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2010-2022. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.deadline.jobrunr;
+
+import org.axonframework.deadline.TestScopeDescriptor;
+import org.axonframework.deadline.jobrunnr.DeadlineDetails;
+import org.axonframework.messaging.ScopeDescriptor;
+import org.jobrunr.utils.mapper.jackson.JacksonJsonMapper;
+import org.junit.jupiter.api.*;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class DeadlineDetailsSerializationTest {
+
+    private static JacksonJsonMapper jacksonJsonMapper;
```

Comment:
The `JacksonJsonMapper` is JobRunr's variant of our `JacksonSerializer`, I assume? Are there other serializers provided by JobRunr too?

Reply:
Yes, Gson, and JacksonB, as they both don't work, I'll think I need to make some changes.

Reply:
Ah gotcha. Looking forward to those changes! :-)

Reply:
Changes are in now, it works even with Jsonb.